src/zepto.js

```diff
@@ -678,6 +678,8 @@ var Zepto = (function() {
         $this.css(props)
       })
       if (!this.length) return null
+      if (!Zepto.contains(document.documentElement, this[0]))
```

Comment:
We should measure which is more expensive: `contains` or `try...catch`, and see if those times are even significant when compared to how long `getBoundingRect()` takes